words.txt

```diff
@@ -114,6 +114,7 @@ inputbytes
 interchangable
 invis
 IPFS
+iSUPC
```

Comment:
I had to add
```suggestion
SUPC
```
instead in https://github.com/ethereum-optimism/specs/pull/651 - is this a typo?

Reply:
either is fine I think, the tool is flexible. The iSUP showed up in my lint report, so I added it as-is.